hidden-role-to-go-core/src/main/java/com/github/virtualgamers/hrtg/core/model/Game.java

```diff
@@ -0,0 +1,30 @@
+package com.github.virtualgamers.hrtg.core.model;
+
+import java.io.Serializable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+import org.hibernate.annotations.GenericGenerator;
+
+/**
+ * {@link Game} to represent a {@link Game}
```

Comment:
I have doubts on the helpfulness of this doc

Reply:
Same. But could not come up with much better.

"Represents a Game which a Party will be playing"?

Reply:
Hmm. Contains game details such as Party playing?